charts/n8n/templates/service.yaml

```
@@ -12,10 +12,16 @@ metadata:
 spec:
   type: {{ default "ClusterIP_" .Values.main.service.type }}
   ports:
+    {{ if .Values.main.service.type | eq "NodePort" -}}
```

```suggestion
    {{- if eq .Values.main.service.type "NodePort" }}
```

Let's not use the pipe notation here